logging-modules/log4j2/src/main/java/com/baeldung/logging/log4j2threadinfo/Log4j2ThreadInfo.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.logging.log4j2ThreadInfo;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.stream.IntStream;
+
+public class Log4j2ThreadInfo {
+    private static final Logger LOGGER = LogManager.getLogger(Log4j2ThreadInfo.class);
```

Comment:
Although a final variable, this object reference is not immutable, that is, it´s not a constant and therefore it should not be in uppercase.